app/zipzip-api-app/src/main/java/org/ssafy/zipzipapiapp/auth/service/AuthService.java

```diff
@@ -25,16 +26,23 @@ public class AuthService {
     private final MemberRepository memberRepository;
 
     @Transactional
-    public TokenResponseDto socialLogin(SocialLoginRequestDto request) {
+    public LoginResponse socialLogin(SocialLoginRequest socialLoginRequest) {
 
-        if (request.code() == null) {
+        if (socialLoginRequest.code() == null) {
             throw new BadRequestException(ERR_MISSING_AUTHORIZATION_CODE);
         }
 
         try {
-            SocialInfoDto socialInfo = kakaoAuthService.getKakaoUserData(request);
-            Member member = findMember(socialInfo);
-            return generateTokens(member);
+            SocialInfoDto socialInfoDto = kakaoAuthService.getKakaoUserData(socialLoginRequest);
+            Member member = findMember(socialInfoDto);
+
+            String newRefreshToken = jwtTokenProvider.generateRefreshToken();
+            String newAccessToken = jwtTokenProvider.generateAccessToken(member.getId());
+            memberRepository.updateRefreshToken(newRefreshToken, member.getId());
+
+            return new LoginResponse(newAccessToken, newRefreshToken, member.getId());
```

Comment:
이 부분이 지금 밑에 정의 되어있는 generateToken 안에 로직과 중복인거 같아서요 ! 아래 메서드 최대한 활용해서 중복 코드 제외하면 좋을 것 같습니다 !

```suggestion
            TokenResponse tokenResponse = generateTokens(member);
            return new LoginResponse(tokenResponse.accessToken, tokenResponse.refreshToken, member.getId());
```